packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -228,6 +228,7 @@ export function Checkout( {
 
 	const classNames = joinClasses( [
 		'composite-checkout',
+		'fs-exclude', // this class is used by FullStory to exclude recording session activity
```

Comment:
I'm not against this change because it's just a classname, but FWIW this feels like it might possibly be the wrong place to add this sort of override.

Even though right now it's only used by calypso and WPCOM, the `@automattic/composite-checkout` package is intended to be used in many different environments; I have it working well in wp-admin, for example, and we might eventually use it for Akismet or Crowdsignal. FullStory isn't a dependency of this package so this introduces a bit of a side-effect. The package is not aware of its environment, and if the bug we're trying to resolve is something in calypso, then we might want to fix it there instead of inside this package.

Reply:
Actually, the more I think about this the more I think it _definitely_ belongs in calypso instead (the `client/` directory of this repo) because components from this package are used in places other than checkout (eg: payment management) and I can imagine that we might not want to affect all places this package might be used.

Reply:
@sirbrillig Is there any risk of the `composite-checkout` class name changing? We're using that classname as a rule to exclude recording activity on the checkout page. Would you have any objections to adding a comment that mentions the classname should not change for privacy rules? CC @danielwrobert

Reply:
> Is there any risk of the composite-checkout class name changing?

It's possible it may change at some point in the future if checkout is changed again but not any time soon. It should be safe to use it. 

> Would you have any objections to adding a comment that mentions the classname should not change for privacy rules? 

If you think it would be helpful, comment additions are totally fine.